Composer/packages/client/src/pages/design/DebugPanel/TabExtensions/RuntimeOutputLog/RuntimeOutputLog.tsx

```diff
@@ -38,33 +108,31 @@ export const RuntimeOutputLog: React.FC<{ projectId: string }> = ({ projectId })
         height: 'calc(100% - 25px)',
         display: 'flex',
         flexDirection: 'column',
-        padding: '15px 24px',
+        padding: '10px 16px',
         fontSize: DEFAULT_FONT_SETTINGS.fontSize,
         fontFamily: DEFAULT_FONT_SETTINGS.fontFamily,
         color: `${NeutralColors.black}`,
         width: 'auto',
         overflowY: 'auto',
         overflowX: 'hidden',
       }}
-      data-testid="Runtime-Output-Logs"
+      data-testid="runtime-output-logs"
     >
-      <div
-        css={{
-          margin: 0,
-          wordBreak: 'break-all',
-          whiteSpace: 'pre-wrap',
-          lineHeight: '20px',
-        }}
-        // eslint-disable-next-line react/no-danger
-        dangerouslySetInnerHTML={createMarkup(runtimeLogs)}
-      />
-      <div
-        css={{
-          color: `${SharedColors.red10}`,
-        }}
-      >
-        <ErrorCallout error={botRuntimeErrors} />
-      </div>
+      {runtimeData.standardOutput && (
+        <div
+          css={{
+            margin: 0,
+            wordBreak: 'break-all',
+            whiteSpace: 'pre-wrap',
+            lineHeight: '20px',
+          }}
+          // eslint-disable-next-line react/no-danger
+          dangerouslySetInnerHTML={createMarkup(runtimeData.standardOutput)}
+          data-testid="runtime-standard-output"
+        />
+      )}
+      {botBuildErrors && <ErrorCallout error={botBuildErrors} />}
```

Comment:
does botBuildErrors ever has no value?

Reply:
It has a value only if there are build errors. For ex LUIS key invalid, publish to LUIS failed.

Reply:
the default value in the recoil state shows an object, meaning it always has a value, when you reset it falls back to that default value right?